File: 1552-build-an-array-with-stack-operations/1552-build-an-array-with-stack-operations.java
class Solution {
    public List<String> buildArray(int[] target, int n) {
        var ans = new ArrayList<String>();
        int i = 0;
        for ( int j = 1; j <= target[target.length - 1] ; j++){
            ans.add("Push");
            if(j != target[i]){
                ans.add("Pop");
            }
            else{
                i++;
            }
        }
        return ans;
    }
}

File: 1552-build-an-array-with-stack-operations/1552-build-an-array-with-stack-operations.py
class Solution:
    def buildArray(self, target: List[int], n: int) -> List[str]:
        s=[]
        for i in range (1,target[-1]+1) :
            s.append("Push")
            if i not in target:
                print(i)
                s.append("Pop")
        
        return s